lib/puppet_x/puppetlabs/aws.rb

```diff
@@ -162,6 +162,14 @@ def elb_client(region = default_region)
         self.class.elb_client(region)
       end
 
+      def self.elbv2_client(region = default_region)
```

Comment:
@zleswomp it's coming along slowly (seasonal holidays getting it the way) the basic's are there for reporting the targetgroup/loadbalancer
feel free to take a look, I've pushed my current code to a branch here :- https://github.com/ctidigital/puppetlabs-aws/tree/feature/elbv2 (this is branched off the codebase for this pull request)